metricbeat/module/windows/perfmon/perfmon.go

```diff
@@ -34,7 +29,7 @@ func init() {
 // multiple fetch calls.
 type MetricSet struct {
 	mb.BaseMetricSet
-	counters   []CounterConfig
+	counters   []CounterConfigGroup
```

Comment:
I see that this in the MetricSet, but do we ever need it outside the New part? It seems to be all in the handle/query anyways?